mkt/search/indexers.py

```diff
@@ -352,7 +358,7 @@ def attach_language_specific_analyzers(cls, mapping, field_names):
         """
         def _locale_field_mapping(field, analyzer):
             return {
-                '%s_%s' % (field, analyzer): {
+                '%s_l_%s' % (field, analyzer): {
```

Comment:
is `l` for l10n? I'm assuming this is to help differentiate the field in some way? Maybe a comment somewhere would help explain the single letter.